skeleton-typescript-webpack/package.json

```diff
@@ -113,6 +113,7 @@
     "@types/webpack": "^2.0.0",
     "@types/whatwg-fetch": "^0.0.32",
     "aurelia-protractor-plugin": "^1.0.1",
+    "babel-polyfill": "^6.16.0",
```

Comment:
Why is the babel-polyfill needed for the typescript skeleton to work?

Also, can you make the same change to the esnext skeleton?